.github/ISSUE_TEMPLATE/docs.yml

```diff
@@ -0,0 +1,16 @@
+name: Docs
+description: Choose this template to create a documentation issue.
+title: "[Docs] Title"
+labels: "area/documentation"
+body:
+- type: textarea
+  id: documentation
+  attributes:
+    label: Description
+    placeholder: |
+      **Provide a detailed description of the information you want to update/include in the YugabyteDB docs.**
+  validations:
+    required: true 
+    
+   
+        
```

Comment:
nit: space

Reply:
(As in, "files should end with a single newline. Remove the rest of the whitespace.")

Reply:
Done!